store/src/java-test/com/zimbra/cs/filter/AddHeaderTest.java

```diff
@@ -499,9 +499,16 @@ public void testAddHeaderNameAsMultipleSpace() {
     @Test
     public void testAddHeaderNameStartingWithSpace() {
         try {
-           String filterScript = "require [\"editheader\"];\n"
-                    + " addheader \" X-My-Test\" \"my-new-header-value\" \r\n"
-                    + "  ;\n";
+//           String filterScript = "require [\"editheader\"];\n"
+//                    + " addheader \" X-My-Test\" \"my-new-header-value\" \r\n"
+//                    + "  ;\n";
```

Comment:
Remove commented lines (or add some explanation the reason why we need to leave the old filterScript)